### PR TITLE
Also publish images from the CAPO release-0.3 branch

### PR DESCRIPTION
**Pull request by sbueringer**
CAPO v0.3.0 has been release now. So let's also publish images from the new release branch

### CODE REVIEW

**Comment by BenTheElder** [comment]
/approve
please prefer LGTM from someone with context on the project first, that way we can mechanically approve if this is OK on the cluster w/o spending cycles on other changes that aren't relevant to us :-)
/lgtm

**Comment by sbueringer** [comment]
> /approve
> please prefer LGTM from someone with context on the project first, that way we can mechanically approve if this is OK on the cluster w/o spending cycles on other changes that aren't relevant to us :-)
> /lgtm

Makes sense, next time, thx 😃

File: config/jobs/image-pushing/k8s-staging-cluster-api.yaml [hunk 1]
```diff
@@ -128,6 +128,7 @@ postsubmits:
       branches:
         - ^master$
         - ^release-0.2$
+        - ^release-0.3$
```
Comment (fejta) [inline config/jobs/image-pushing/k8s-staging-cluster-api.yaml]: release-.+?

**Comment by sbueringer** [comment]
@fejta fixed. I need a new /lgtm :)

**Pull request merged**